source/renderer/app/stores/StakingStore.js

```diff
@@ -70,7 +95,9 @@ export default class StakingStore extends Store {
   @observable calculateDelegationFeeRequest: Request<BigNumber> = new Request(
     this.api.ada.calculateDelegationFee
   );
-  @observable isStakingExperimentRead: boolean = false;
+  @observable submitRedeemItnRewardsRequest: Request<any> = new Request(
```

Comment:
@daniloprates Can you add TODO flag here for Requst type so we can update once API endpoint is ready and remove `any`